lesson17/main.js

```diff
@@ -0,0 +1,75 @@
+const jsonUrl = "https://jsondata.okiba.me/v1/json/YbIZe211030061143";
+const wrapper = document.getElementById("js-wrapper");
+const ul = document.getElementById("js-slideshow");
+const numOfSecond = 3;
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const createLoader = () => {
+  const loader = document.createElement("div");
+  const loaderImage = document.createElement("img");
+  loader.id = "js-loader";
+  loaderImage.src = "img/loading-circle.gif";
+  wrapper.appendChild(loader).appendChild(loaderImage);
+};
+
+const loading = () => {
+  const loader = document.getElementById("js-loader");
+  loader.classList.add("loading");
+};
+
+const loaded = () => {
+  const loader = document.getElementById("js-loader");
+  loader.classList.add("loaded");
```

Comment:
I found this code interesting as I just did "removeChild" to hide the loading image. Great work!

Reply:
Thank you for reviewing.
When I searched making the loading screen before, I learned this way. So I implemented the same way this time.But removing sounds more simple and easy.
Next time I will do the way you taught me.